### PR TITLE
doc: Update README.md with slack invite and channel links

### LINKED ISSUE #4391: Invite link to slack workspace is not active
# Describe the bug #

invite link to slack channel is broken.

# Environment details #

N/A this is in the readme of this repo

# Steps to reproduce #

1. open [https://bit.ly/40FQu7u](https://bit.ly/40FQu7u) a browser
# Actual results #

Slack error page saying 
```
This link is no longer active
```

# Expected behavior #

Expected to join a new slack workspace

# Logs #

N/A

# Additional context #

nope.

### PR DESCRIPTION
**Pull request by switchboardOp**
# Describe what this PR does #

This MR updates the slack invite link in `README.md` with one provided by @nixpanic on the related issue and includes a direct link to the `#ceph-csi` channel therein.

## Is there anything that requires special attention ##

nope

## Related issues ##

Fixes: #4391

## Future concerns ##

N/A

**Checklist:**

* [x] **Commit Message Formatting**: Commit titles and messages follow
  guidelines in the [developer
  guide](https://github.com/ceph/ceph-csi/blob/devel/docs/development-guide.md#commit-messages).
* [x] Reviewed the developer guide on [Submitting a Pull
  Request](https://github.com/ceph/ceph-csi/blob/devel/docs/development-guide.md#development-workflow)
* [ ] [Pending release
  notes](https://github.com/ceph/ceph-csi/blob/devel/PendingReleaseNotes.md)
  updated with breaking and/or notable changes for the next major release.
* [ ] Documentation has been updated, if necessary.
* [ ] Unit tests have been added, if necessary.
* [ ] Integration tests have been added, if necessary.

---

<details>
<summary>Show available bot commands</summary>

These commands are normally not required, but in case of issues, leave any of
the following bot commands in an otherwise empty comment in this PR:

* `/retest ci/centos/<job-name>`: retest the `<job-name>` after unrelated
  failure (please report the failure too!)

</details>

### CODE REVIEW

**Review by karthik-us — APPROVED** [review]
LGTM.

**Review by yati1998 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -199,8 +199,8 @@ welcome and encouraged to join.
 
 Please use the following to reach members of the community:
 
-- Slack: Join our [Slack channel](https://ceph-storage.slack.com) to discuss
+- Slack: Join the [#ceph-csi](https://ceph-storage.slack.com/archives/C05522L7P60) channel on the [ceph Slack](https://ceph-storage.slack.com) to discuss
```
Comment (nixpanic) [inline README.md]: Thanks for the PR!

Unfortunately `yamllint` fails because this line is now too long. Can you add line-breaks before 80 characters?

Comment (switchboardOp) [reply README.md]: np. lines should be sufficiently short now.

File: README.md [hunk 2]
```diff
@@ -199,8 +199,10 @@ welcome and encouraged to join.
...
 Please use the following to reach members of the community:
 
-- Slack: Join our [Slack channel](https://ceph-storage.slack.com) to discuss
-  anything related to this project. You can join the Slack by this [invite
-  link](https://bit.ly/40FQu7u)
+- Slack: Join the
+  [#ceph-csi](https://ceph-storage.slack.com/archives/C05522L7P60) channel 
+  on the [ceph Slack](https://ceph-storage.slack.com) to discuss anything
+  related to this project. You can join the Slack by this
+  [invite link](bit.ly/ceph-slack-invite)
```
Comment (nixpanic) [inline README.md:206]: Thanks!

For any others that wonder how to verify the link, you can add a `+` to the URL, like `https://bit.ly/ceph-slack-invite+` to see where it points to.

**Review by nixpanic — DISMISSED** [review]

**Review by yati1998 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
@mergifyio rebase

**Pull request merged**